addons/snailmail/static/tests/message/message_patch.test.js

```diff
@@ -186,25 +159,12 @@ test("Trial Error", async () => {
         notification_type: "snail",
         res_partner_id: partnerId,
     });
-    const def = new Deferred();
-    onRpc("mail.message", "send_letter", (args) => {
-        if (args.args[0][0] === messageId) {
-            asyncStep(args.method);
-            def.resolve();
-        }
-        return true;
-    });
     await start();
     await openFormView("res.partner", partnerId);
     await click(".o-mail-Message-notification i.fa-paper-plane");
-    await contains(".o-snailmail-SnailmailError p", {
-        text: "You need credits on your IAP account to send a letter.",
+    await contains(".o-snailmail-SnailmailNotificationPopover", {
+        text: "(Snailmail Trial Error)",
```

Comment:
In my opinion, the error messages are not very clear. Here, for instance, when I get the error message: "(Snailmail Trial Error)", it's not clear to me that I have not enough credit on my IAP account and that I need to purchase new credits.

In the `failureMessage` getter of `notificationPatch`, we can maybe provide more precise error messages and give an indication on how to solve the errors?